src/VirtoCommerce.ImageToolsModule.Core/ModuleConstants.cs

```diff
@@ -57,6 +60,19 @@ public static class General
                     DefaultValue = 50,
                 };
 
+                public static SettingDescriptor AllowedImageFormats { get; } = new()
```

Comment:
In other modules we use static readonly fields (not properties). Perhaps we need to think about refactoring this part of the code.

Reply:
No, we prefer properties